diff --git a/node_modules/sqlite3/deps/sqlite3.gyp b/node_modules/sqlite3/deps/sqlite3.gyp
index 61b3085..dbdc9ef 100755
--- a/node_modules/sqlite3/deps/sqlite3.gyp
+++ b/node_modules/sqlite3/deps/sqlite3.gyp
@@ -60,7 +60,7 @@
           'outputs': [
             '<(SHARED_INTERMEDIATE_DIR)/sqlite-autoconf-<@(sqlite_version)/sqlite3.c'
           ],
-          'action': ['<!(node -p "process.env.npm_config_python || \\"python\\"")','./extract.py','./sqlite-autoconf-<@(sqlite_version).tar.gz','<(SHARED_INTERMEDIATE_DIR)']
+          'action': ['<!(node -p "process.env.npm_config_python || \\"python3\\"")','./extract.py','./sqlite-autoconf-<@(sqlite_version).tar.gz','<(SHARED_INTERMEDIATE_DIR)']
         }
       ],
       'direct_dependent_settings': {
